proto/v1/risk_service.proto

```diff
@@ -0,0 +1,120 @@
+syntax = "proto3";
+
+package bytebase.v1;
+
+import "google/api/annotations.proto";
+import "google/api/client.proto";
+import "google/api/expr/v1alpha1/syntax.proto";
+import "google/api/field_behavior.proto";
+import "google/protobuf/empty.proto";
+import "google/protobuf/field_mask.proto";
+
+option go_package = "generated-go/v1";
+
+service RiskService {
+  rpc ListRisks(ListRisksRequest) returns (ListRisksResponse) {
+    option (google.api.http) = {get: "/v1/risks"};
+    option (google.api.method_signature) = "";
+  }
+
+  rpc UpdateRisk(UpdateRiskRequest) returns (Risk) {
+    option (google.api.http) = {
+      patch: "/v1/{risk.name=risks/*}"
+      body: "risk"
+    };
+    option (google.api.method_signature) = "risk,update_mask";
+  }
+
+  rpc BatchUpdateRisks(BatchUpdateRisksRequest) returns (BatchUpdateRisksResponse) {
```

Comment:
Do we need BatchUpdate at this moment?

And do we need Get/Delete?

Reply:
We do need BatchUpdate because we need to update 2 risks at the same time when transferring a risk rule to another risk.

Get/Delete is not needed because we only have predefined risks at the moment